e2etest/l2tests/assignment.go

```diff
@@ -208,17 +209,18 @@ var _ = ginkgo.Describe("IP Assignment", func() {
 					AllocateTo: &metallbv1beta1.ServiceAllocation{Priority: 10, NamespaceSelectors: []metav1.LabelSelector{{MatchLabels: f.Namespace.Labels}}},
 				},
 			}
-			defaultPool := metallbv1beta1.IPAddressPool{
+			defaultNsPool := metallbv1beta1.IPAddressPool{
```

Comment:
same, "namespacePoolWithNoPriority", also same for the ones below. I'd try to be consistent with the name